File: CS/PersonalOrganizer/ContactContextDataModel/DesignTime/DesignTimeContactRepository.cs
using System;
using System.Linq;
using System.Data;
using System.Data.Entity;
using System.Linq.Expressions;
using System.Collections.Generic;
using PersonalOrganizer.Common.Utils;
using PersonalOrganizer.Model;
using PersonalOrganizer.Common.DataModel;
using PersonalOrganizer.Common.DataModel.EntityFramework;

namespace PersonalOrganizer.ContactContextDataModel {
    public class DesignTimeContactRepository : DesignTimeRepository<Contact, int>, IContactRepository {
        public DesignTimeContactRepository()
            : base(x => x.Id) {
        }
    }
}

File: VB/PersonalOrganizer/ContactContextDataModel/DesignTime/DesignTimeContactRepository.vb
Imports System
Imports System.Linq
Imports System.Data
Imports System.Data.Entity
Imports System.Linq.Expressions
Imports System.Collections.Generic
Imports Common.Utils
Imports PersonalOrganizer.Model
Imports Common.DataModel
Imports Common.DataModel.EntityFramework
Namespace ContactContextDataModel
    Public Class DesignTimeContactRepository
        Inherits DesignTimeRepository(Of Contact, Integer)
        Implements IContactRepository
        Public Sub New()
            MyBase.New(Function(x) x.Id)
        End Sub
    End Class
End Namespace
